draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1709,20 +1709,20 @@ Steps:
 
 ~~~
 t = {}
-u = LE64(data_len_bits) || LE64(tag_length_bits)
+u = LE64(data_len_bits) || LE64(tag_len_bits)
 for i in 0..D:
     t = t || (V[2,i] ^ u)
 
 Repeat(7, Update(t, t))
 
 tags = {}
-if tag_length_bits == 128:
-    for i in 0..D:   # tag from state 0 is included
+if tag_len_bits == 128:
+    for i in 0..D:  # tag from state 0 is included
         ti = V[0,i] ^ V[1,i] ^ V[2,i] ^ V[3,i] ^ V[4,i] ^ V[5,i] ^ V[6,i]
         tags = tags || ti
 
-else:                # 256 bits
-    for i in 1..D:   # tag from state 0 is skipped
+else:              # 256 bits
```

Comment:
Think these comments need to be shifted right to match the top one.

Reply:
The top one has been moved to the left.